### ISSUE TITLE
Print behaviour & test requirements

### ISSUE THREAD

**Issue by gsnedders** [w3c/css-houdini-drafts#978]
Does anywhere normatively say what should happen when you print the document? I guess it's technically implementation defined, and just assumed that the page is laid out for the new medium (and print medium styles are applied)?

Given historically, JS is not run when redoing style & layout for print, what do we expect the behaviour with the Layout API to be? Do we expect the registered JS to be called? What happens if it doesn't halt (do you fallback to breaking the existing screen rendering, or do we just give a script hang prompt, etc.)?

How do we expect to test such behaviour? Are we just envisioning [print reftests](https://github.com/web-platform-tests/wpt/issues/7760) for this, or would there be in any way the worklets would be tested with testharness.js?

**Comment by tabatkins** [comment]
Hmm, I wonder what spec this would go in. I suppose it isn't a general Worklets thing; Audio worklets won't run on print. I suppose it's some boilerplate I'd just need to spam into each of the relevant specs.

I think it needs to go in the Introduction, like the Module Interactions that some specs have.

**Comment by gsnedders** [comment]
We certainly have some normative requirements here, no? Do we not need to call out explicitly that the layout worklet should be run when going from scrolling media to paged media, for example when printing from an on-screen browser?

**Comment by tabatkins** [comment]
Yes, I'm putting an indicator on it that it's a normative section.
